Nov 04 2008	Arnaud Charlet	<charlet@adacore.com>

	* glade/glade_menu_editor.c, glade/glade_project_view.h,
	* glade/glade_keys_dialog.c, glade/gbwidgets/gbclist.c: Fix compilation
	error with Gtk+ 2.14.4

Fri Sep 29 2006	Arnaud Charlet	<charlet@adacore.com>

	* glade/glade_project_options.c: Fix invalid C syntax now detected
	by GCC 4.1

Tue Jun 17 2003	Arnaud Charlet	<charlet@act-europe.fr>

	* glade/utils.c: Remove wrong/unneeded Win32 specific code.

Mon Jun 16 2003	Arnaud Charlet	<charlet@act-europe.fr>

	* glade/main.c: Disable gail and non portable nls calls.
	* glade/glade_project.c: Set Ada as the default language.
	* configure.in: Disable check for scrollkeeper, not needed/available.

Thu Jun 12 2003	Nicolas Setton	<setton@act-europe.fr>

	* glade/glade_project.c: Generate Ada as well as C and C++.

--- glade/main.c.old	Mon Jun 16 15:53:41 2003
+++ glade/main.c	Mon Jun 16 16:07:16 2003
@@ -122,7 +122,7 @@
 #ifdef USE_GNOME
   modules_needed = "gail"; /* FIXME: Add gail-gnome when it is ready. */
 #else
-  modules_needed = "gail";
+  modules_needed = "";
 #endif
   new_modules = g_strdup_printf ("GTK_MODULES=%s%s%s",
 				 modules ? modules : "",
@@ -173,7 +173,7 @@
 
 #ifdef ENABLE_NLS
   bindtextdomain (GETTEXT_PACKAGE, GLADE_LOCALEDIR);
-  bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
+  /* bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8"); */
   textdomain (GETTEXT_PACKAGE);
 #endif
 
--- glade/glade_project.c.old	Mon Jun 16 16:02:15 2003
+++ glade/glade_project.c	Mon Jun 16 16:11:07 2003
@@ -46,7 +46,7 @@
 /* The order must match the GladeLanguageType enum in glade_project.h. */
 gchar *GladeLanguages[] = { "C", "C++" , "Ada 95", "Perl", "Eiffel" };
 #if 1
-gint GladeNumLanguages = 2; /* Only C and C++ for now. */
+gint GladeNumLanguages = 3; /* Only C, C++ and Ada for now. */
 #else
 gint GladeNumLanguages = sizeof (GladeLanguages) / sizeof (GladeLanguages[0]);
 #endif
@@ -160,7 +160,7 @@
   project->source_directory = NULL;
   project->pixmaps_directory = NULL;
 
-  project->language = GLADE_LANGUAGE_C;
+  project->language = GLADE_LANGUAGE_ADA95;
   property_show_lang_specific_page (project->language);
 
   project->changed = FALSE;
--- configure.in.old	Mon Jun 16 15:45:38 2003
+++ configure.in	Mon Jun 16 15:45:39 2003
@@ -114,18 +114,6 @@
 
 
 dnl ***************************************************************************
-dnl Check for ScrollKeeper
-dnl ***************************************************************************
-
-AC_PATH_PROG(SCROLLKEEPER_CONFIG, scrollkeeper-config,no)
-if test "$SCROLLKEEPER_CONFIG" = "no"; then
-  AC_MSG_ERROR(Couldn't find scrollkeeper-config, please install the scrollkeeper package)
-fi
-SCROLLKEEPER_REQUIRED=0.1.4
-AC_SUBST(SCROLLKEEPER_REQUIRED)
-
-
-dnl ***************************************************************************
 dnl Debugging - defines GLADE_DEBUG if --enable-debug option is used.
 dnl ***************************************************************************
 
--- configure.old	Fri Apr 11 14:37:55 2003
+++ configure	Thu Jun 15 15:00:04 2006
@@ -11925,56 +11925,6 @@ fi
 
 
 
-# Extract the first word of "scrollkeeper-config", so it can be a program name with args.
-set dummy scrollkeeper-config; ac_word=$2
-echo "$as_me:$LINENO: checking for $ac_word" >&5
-echo $ECHO_N "checking for $ac_word... $ECHO_C" >&6
-if test "${ac_cv_path_SCROLLKEEPER_CONFIG+set}" = set; then
-  echo $ECHO_N "(cached) $ECHO_C" >&6
-else
-  case $SCROLLKEEPER_CONFIG in
-  [\\/]* | ?:[\\/]*)
-  ac_cv_path_SCROLLKEEPER_CONFIG="$SCROLLKEEPER_CONFIG" # Let the user override the test with a path.
-  ;;
-  *)
-  as_save_IFS=$IFS; IFS=$PATH_SEPARATOR
-for as_dir in $PATH
-do
-  IFS=$as_save_IFS
-  test -z "$as_dir" && as_dir=.
-  for ac_exec_ext in '' $ac_executable_extensions; do
-  if $as_executable_p "$as_dir/$ac_word$ac_exec_ext"; then
-    ac_cv_path_SCROLLKEEPER_CONFIG="$as_dir/$ac_word$ac_exec_ext"
-    echo "$as_me:$LINENO: found $as_dir/$ac_word$ac_exec_ext" >&5
-    break 2
-  fi
-done
-done
-
-  test -z "$ac_cv_path_SCROLLKEEPER_CONFIG" && ac_cv_path_SCROLLKEEPER_CONFIG="no"
-  ;;
-esac
-fi
-SCROLLKEEPER_CONFIG=$ac_cv_path_SCROLLKEEPER_CONFIG
-
-if test -n "$SCROLLKEEPER_CONFIG"; then
-  echo "$as_me:$LINENO: result: $SCROLLKEEPER_CONFIG" >&5
-echo "${ECHO_T}$SCROLLKEEPER_CONFIG" >&6
-else
-  echo "$as_me:$LINENO: result: no" >&5
-echo "${ECHO_T}no" >&6
-fi
-
-if test "$SCROLLKEEPER_CONFIG" = "no"; then
-  { { echo "$as_me:$LINENO: error: Couldn't find scrollkeeper-config" >&5
-echo "$as_me: error: Couldn't find scrollkeeper-config" >&2;}
-   { (exit please install the scrollkeeper package); exit please install the scrollkeeper package; }; }
-fi
-SCROLLKEEPER_REQUIRED=0.1.4
-
-
-
-
 # Check whether --enable-debug or --disable-debug was given.
 if test "${enable_debug+set}" = set; then
   enableval="$enable_debug"
@@ -12668,8 +12618,6 @@ s,@INTLOBJS@,$INTLOBJS,;t t
 s,@POFILES@,$POFILES,;t t
 s,@POSUB@,$POSUB,;t t
 s,@MKINSTALLDIRS@,$MKINSTALLDIRS,;t t
-s,@SCROLLKEEPER_CONFIG@,$SCROLLKEEPER_CONFIG,;t t
-s,@SCROLLKEEPER_REQUIRED@,$SCROLLKEEPER_REQUIRED,;t t
 CEOF
 
 _ACEOF
--- glade/utils.c.old	2003-06-17 14:26:50.000000000 +0100
+++ glade/utils.c	2003-06-17 14:26:57.000000000 +0100
@@ -21,13 +21,8 @@
 #include <string.h>
 #include <sys/types.h>
 #include <sys/stat.h>
-#ifdef _WIN32
-#include <ctype.h>
-#include <direct.h>
-#else
 #include <unistd.h>
 #include <dirent.h>
-#endif
 #include <errno.h>
 
 #include <gtk/gtk.h>
--- glade/glade_project_options.c.old	2003-04-02 08:56:17.000000000 -0500
+++ glade/glade_project_options.c	2006-09-29 04:06:54.543132935 -0400
@@ -1074,7 +1074,7 @@ glade_project_options_generate_program_n
 {
   gchar *project_name, *program_name;
 
-  (char*) project_name = gtk_entry_get_text (GTK_ENTRY (options->name_entry));
+  project_name = (gchar *)gtk_entry_get_text (GTK_ENTRY (options->name_entry));
   program_name = glade_project_options_generate_program_name_from_project_name (project_name);
   options->auto_generation_level++;
   gtk_entry_set_text (GTK_ENTRY (options->program_name_entry), program_name);
--- glade/gbwidgets/gbclist.c.old	2008-11-04 15:06:42.000000000 +0100
+++ glade/gbwidgets/gbclist.c	2008-11-04 15:06:45.000000000 +0100
@@ -19,6 +19,7 @@
 
 #include <string.h>
 
+#include <gtk/gtkctree.h>
 #include <gtk/gtkclist.h>
 #include <gtk/gtkhbox.h>
 #include <gtk/gtkmain.h>
--- glade/glade_keys_dialog.c.old	2008-11-04 15:21:27.000000000 +0100
+++ glade/glade_keys_dialog.c	2008-11-04 15:21:46.000000000 +0100
@@ -25,6 +25,7 @@
 
 #include <gtk/gtkbox.h>
 #include <gtk/gtkbutton.h>
+#include <gtk/gtkctree.h>
 #include <gtk/gtkclist.h>
 #include <gtk/gtkstock.h>
 #include <gtk/gtkscrolledwindow.h>
--- glade/glade_menu_editor.c.old	2008-11-04 15:36:26.000000000 +0100
+++ glade/glade_menu_editor.c	2008-11-04 15:36:39.000000000 +0100
@@ -28,6 +28,7 @@
 #include <gdk/gdkkeysyms.h>
 #include <gtk/gtkarrow.h>
 #include <gtk/gtkaccellabel.h>
+#include <gtk/gtkctree.h>
 #include <gtk/gtkclist.h>
 #include <gtk/gtkcombo.h>
 #include <gtk/gtkentry.h>
--- glade/glade_project_view.h.old	2008-11-04 15:37:08.000000000 +0100
+++ glade/glade_project_view.h	2008-11-04 15:37:17.000000000 +0100
@@ -18,6 +18,7 @@
 #ifndef GLADE_PROJECT_VIEW_H
 #define GLADE_PROJECT_VIEW_H
 
+#include <gtk/gtkctree.h>
 #include <gtk/gtkclist.h>
 
 #include "glade_project.h"
